coda_v2_python_client/firebase_client_wrapper.py

```diff
@@ -525,20 +525,20 @@ def compute_segment_messages_metrics(self, segment_id, messages=None):
 
         return MessagesMetrics(len(messages), messages_with_labels, wrong_scheme_messages, not_coded_messages)
 
-    def compute_coding_progress(self, dataset_id):
+    def compute_messages_metrics(self, dataset_id):
```

Comment:
Consider
```suggestion
    def compute_and_update_messages_metrics(self, dataset_id):
```
since this does more than just compute the metrics

Reply:
It only computes the metrics. I've updated the function to return the metrics.